react/legacy/components/TopMenu.js

```diff
@@ -23,7 +23,7 @@ const TopMenu = ({
   collapsibleAnimation,
   mobile,
 }) => {
-  const hasMenu = !!useChildBlock__unstable({id: 'unstable--menu'})
+  const hasMenu = !!useChildBlock({ id: 'menu' })
```

Comment:
You need this `!!` for what purpose?

Reply:
It returns an object, he wants a boolean